### DISCUSSION TITLE
create-docz-app script broken?

### DISCUSSION THREAD

**Discussion by 1yzo** [pedronauck/docz#1644] [Q&A]
Is `npx create-docz-app` working for anyone? I've tried using different versions of node as well but I'm getting this error everytime.

![image](https://user-images.githubusercontent.com/33081256/125681152-7082c180-b7b0-46b0-9813-55a71eb30bc4.png)

**Comment by renatobenks** [comment]
hey @1yzo, it seems it is! But unfortunately, this is not official from docz, once we're still trying to understand if it does make sense to the community has this package because usually you start with docz on an existent application. Let me know your thoughts and what you think.

Anyways, I'll straight this clear on the README.md, also try to follow up on this issue https://github.com/doczjs/docz/issues/1452
